bftengine/src/bftengine/messages/InternalMessage.hpp

```diff
@@ -24,6 +24,7 @@
 #include "messages/PrePrepareCarrierInternalMsg.hpp"
 #include "messages/ValidatedMessageCarrierInternalMsg.hpp"
 #include "messages/FinishPrePrepareExecutionInternalMsg.hpp"
+#include "messages/ClientReplyMsg.hpp"
```

Comment:
Where do you use clientReplyMsg as internal?

Reply:
fixed